config/secrets.exs.example

```diff
@@ -22,3 +22,16 @@ config :cadet,
 
 config :sentry,
   dsn: "https://public_key/sentry.io/somethingsomething"
+
+# Configure Arc File Upload for development only
+
+# Uncomment below to use local storage
+config :arc, storage: Arc.Storage.Local
+
+# Uncomment below to use AWS S3
+# config :arc,
+#   virtual_host: true
+#
+# config :ex_aws,
+#   access_key_id: "ACCESS_KEY_ID",
```

Comment:
its really bad practice to have aws secret keys in config files if youre running on ec2 prefer to use the ec2 instance profile/role instead.

Refer to their documentation to see how they do it!
https://github.com/ex-aws/ex_aws

Reply:
They are not meant for production. Move to `dev.secrets.exs.example`.